applications/FluidDynamicsApplication/custom_elements/fluid_element.cpp

```diff
@@ -807,6 +808,12 @@ void StrainRateSpecialization<TElementData,3>::Calculate(
 template class FluidElement< SymbolicNavierStokesData<2,3> >;
 template class FluidElement< SymbolicNavierStokesData<3,4> >;
 
+template class FluidElement< SymbolicStokesData<2,3> >;
+template class FluidElement< SymbolicStokesData<2,4> >;
+template class FluidElement< SymbolicStokesData<3,4> >;
+template class FluidElement< SymbolicStokesData<3,6> >;
```

Comment:
Why `<3,6>`? Does it makes sense to have wedge elements in the context of CFD?

Reply:
I agree with @rubenzorrilla 
Wedge elements are (_usually_) utilized when directionality matters, e.g. modeling shells, or some Darcy flow cases.

Reply:
it is also positive for Boundary Layers, so i think the prism is an important element

Reply:
OK. Forget about my comment @mohittyagi-k and my apologies for the noise. Thanks for the explanation @RiccardoRossi .